application-tour-ui/src/main/resources/TourCode/StepsListing.xml

```diff
@@ -101,7 +100,9 @@
                      $!placement
                   &lt;/td&gt;
                   &lt;td&gt;
-                     $!targetPage
+                     $!targetPage &lt;br&gt;
+                     $!action &lt;br&gt;
+                     $!queryString
```

Comment:
What if the page is Space.Page? You'd end up with `edit/Space.Page?query` instead of `edit/Space/Page?query`, which seems a bit confusing to me.